Javascript/variables.js

```diff
@@ -3,69 +3,71 @@ var runeexpbase = [1, 4, 9, 16, 1000]
 
 // this shows the logarithm of costs. ex: upgrade one will cost 1e+6 coins, upgrade 2 1e+7, etc.
 var upgradeCosts = [0, 6, 7, 8, 10, 12, 20, 25, 30, 35, 45, 55, 75, 110, 150, 200, 250, 500, 750, 1000, 1500,

```

Comment:
Change to `const` as these are never reassigned.

Reply:
let and const stops the variable from being global, and in this case it is global, so it has to be var.